modules/shared/src/main/scala/io/constellationnetwork/schema/address.scala

```diff
@@ -24,8 +24,8 @@ object address {
   object Address {
 
     def fromBytes(bytes: Array[Byte]): Address = {
-      val hashCode = Hash.hashCodeFromBytes(bytes)
-      val encoded = Base58.encode(hashCode.asBytes().toIndexedSeq)
+      val hashCode = Hash.sha256FromBytes(bytes)
```

Comment:
Is it still a `hashCode`? You've changed the function name but not a variable

Reply:
It's not an instance of `HashCode`, it's a byte array that contains a SHA-256 digest.

Reply:
Ok, so you should change the `hashCode` variable name too